initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -83,87 +85,98 @@ while true ; do
             shift ;;
         -e|--env) leaveenv=1
             shift ;;
+        -p|--purge) purge='purge'
+            shift ;;
         -h|--help) usage
                    shift ;;
         --) shift ; break ;;
         *) usage ;;
     esac
 done
 
-set_environmentvars
+pckname=$(apt-cache pkgnames hpccsystems-platform 2> /dev/null)
 
-mklink=""
-lpath=$(canonicalize_path ${path})
-if [ -n "${lpath}" -a -h "${lpath}" ] ; then
-    mklink=$(readlink "${lpath}")
-    # echo "\"${lpath}\" is a soft-link to \"${mklink}\""
+if [ $purge != 'purge' ];then
```

Comment:
This line needs to be [ "$purge" != "purge" ]  or else you'll get a unary operator error for the != if purge isn't populated.